src/commands/install-deps.yml

```diff
@@ -9,34 +9,32 @@ parameters:
     type: string
     default: "gems-v1"
   path:
-    description: "Installation path."
+    description: "Project path."
     default: "."
     type: string
-  bundler-version:
```

Comment:
Maybe we can leave it as an override option. WDYT?

Reply:
I think it's a good idea to have it as an override, especially if the command to determine the version from the lock file fails for whatever reason.

Reply:
Done! Thank you!